.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,10 +1,40 @@
-## What changes were proposed in this pull request?
+<!--
+Thanks for sending a pull request!  Here are some tips for you:
+  1. If this is your first time, please read our contributor guidelines: https://spark.apache.org/contributing.html
+  2. Ensure you have added or run the appropriate tests for your PR: https://spark.apache.org/developer-tools.html
+  3. If the PR is unfinished, add '[WIP]' in your PR title, e.g., '[WIP][SPARK-XXXX] Your PR title ...'.
+  4. Update your PR description whenever it is needed.
+  5. Provide a concise example to reproduce the issue if possible for a faster review.
```

Comment:
Is this necessary? There should be a unit test to show the issue, but if not then a concise example should be provided

Reply:
Nope, it's not necessary but just for a faster review. None of them here are strictly required but rather guidelines.